Removes Sphinx from the build as it is not required
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/CellBender-0.2.1.orig/REQUIREMENTS.txt b/CellBender-0.2.1/REQUIREMENTS.txt
index 535e3e7..0646183 100644
--- a/CellBender-0.2.1.orig/REQUIREMENTS.txt
+++ b/CellBender-0.2.1/REQUIREMENTS.txt
@@ -6,8 +6,3 @@ pandas
 pyro-ppl>=0.3.2
 scikit-learn
 matplotlib
-sphinx>=2.1
-sphinx-rtd-theme
-sphinx-autodoc-typehints
-sphinxcontrib-programoutput
-sphinx-argparse
